torch/csrc/distributed/rpc/rref_proto.h

```diff
@@ -82,31 +98,49 @@ class TORCH_API PythonRRefFetchCall final : public RRefMessageBase {
 // OwnerRRef uses this message to send the RRef value to a remote UserRRef
 class TORCH_API RRefFetchRet : public RpcCommandBase {
  public:
-  RRefFetchRet(std::vector<at::IValue> values, MessageType type)
-      : values_(std::move(values)), type_(type) {}
+  RRefFetchRet(
+      std::vector<at::IValue> values,
+      MessageType type,
+      DeviceMap&& deviceMap)
```

Comment:
does the `RRefFetchRet` need a deviceMap?

Reply:
`rref.to_here(device_map)` propagates `device_map` to `ScriptRRefFetchRet`/`PythonRRefFetchRet` to put the result to the target device